common-api/src/main/java/org/pmiops/workbench/cdr/dao/DomainInfoDao.java

```diff
@@ -0,0 +1,51 @@
+package org.pmiops.workbench.cdr.dao;
+
+import java.util.List;
+import org.pmiops.workbench.cdr.model.DomainInfo;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+
+public interface DomainInfoDao extends CrudRepository<DomainInfo, Long> {
+
+  // TODO: consider not using @Query since it doesn't let us re-use shared SQL expressions for these methods?
+
+  /**
+   * Returns domain metadata and concept counts for domains, matching standard concepts by name
+   * and all concepts by code or concept ID.
+   */
+  @Query(nativeQuery=true,value="select d.domain, d.name, d.description,\n" +
+      "d.concept_id, COUNT(DISTINCT c.concept_id) as all_concept_count,\n" +
+      "SUM(c.standard_concept IN ('S', 'C')) as standard_concept_count,\n" +
```

Comment:
This is a basic question, but what is the format of this column and where is it documented? I was sort of expecting that 'S' and 'C' were the only legal values, but then I'm not sure this select statement makes sense.

Reply:
'S' = standard. 'C'=  standard classification... meaning it's like a standard higher-level concept, which may or may not have any rows in the data directly... (instead it may just have descendant concepts based on relationships with rows in the data.) We want to match either of those.

Not set is the other valid value for this column, and it means non-standard concept.

Documentation for this is here:

http://www.ohdsi.org/web/wiki/doku.php?id=documentation:cdm:concept